src/addresses.rs

```diff
@@ -97,396 +151,478 @@ impl Default for ChainAddresses {
-        )),
-        tick_lens_address: Some(address!("b52429333da969a0C79a60930a4Bf0020E5D1DE8")),
-        swap_router02_address: None,
-        v1_mixed_route_quoter_address: None,
-    }
-}
+pub const ARBITRUM_GOERLI_ADDRESSES: ChainAddresses = ChainAddresses {
+    v3_core_factory_address: address!("4893376342d5D7b3e31d4184c08b265e5aB2A3f6"),
+    multicall_address: address!("8260CB40247290317a4c062F3542622367F206Ee"),
+    quoter_address: address!("1dd92b83591781D0C6d98d07391eea4b9a6008FA"),
+    v3_migrator_address: Some(address!("A815919D2584Ac3F76ea9CB62E6Fd40a43BCe0C3")),
+    nonfungible_position_manager_address: Some(address!(
+        "622e4726a167799826d1E1D150b076A7725f5D81"
+    )),
+    tick_lens_address: Some(address!("b52429333da969a0C79a60930a4Bf0020E5D1DE8")),
+    swap_router02_address: None,
+    v1_mixed_route_quoter_address: None,
+};
+
+/// Arbitrum sepolia v3 addresses
+pub const ARBITRUM_SEPOLIA_ADDRESSES: ChainAddresses = ChainAddresses {
+    v3_core_factory_address: address!("248AB79Bbb9bC29bB72f7Cd42F17e054Fc40188e"),
+    multicall_address: address!("2B718b475e385eD29F56775a66aAB1F5cC6B2A0A"),
+    quoter_address: address!("2779a0CC1c3e0E44D2542EC3e79e3864Ae93Ef0B"),
+    v3_migrator_address: Some(address!("398f43ef2c67B941147157DA1c5a868E906E043D")),
+    nonfungible_position_manager_address: Some(address!(
+        "6b2937Bde17889EDCf8fbD8dE31C3C2a70Bc4d65"
+    )),
+    tick_lens_address: Some(address!("b52429333da969a0C79a60930a4Bf0020E5D1DE8")),
+    swap_router02_address: Some(address!("101F443B4d1b059569D643917553c771E1b9663E")),
+    v1_mixed_route_quoter_address: None,
+};
 
 /// sepolia v3 addresses
-pub fn sepolia_address() -> ChainAddresses {
-    ChainAddresses {
-        v3_core_factory_address: address!("0227628f3F023bb0B980b67D528571c95c6DaC1c"),
-        multicall_address: address!("D7F33bCdb21b359c8ee6F0251d30E94832baAd07"),
-        quoter_address: address!("Ed1f6473345F45b75F8179591dd5bA1888cf2FB3"),
-        v3_migrator_address: Some(address!("729004182cF005CEC8Bd85df140094b6aCbe8b15")),
-        nonfungible_position_manager_address: Some(address!(
-            "1238536071E1c677A632429e3655c799b22cDA52"
-        )),
-        tick_lens_address: Some(address!("d7f33bcdb21b359c8ee6f0251d30e94832baad07")),
-        swap_router02_address: None,
-        v1_mixed_route_quoter_address: None,
-    }
-}
+pub const SEPOLIA_ADDRESS: ChainAddresses = ChainAddresses {
+    v3_core_factory_address: address!("0227628f3F023bb0B980b67D528571c95c6DaC1c"),
+    multicall_address: address!("D7F33bCdb21b359c8ee6F0251d30E94832baAd07"),
+    quoter_address: address!("Ed1f6473345F45b75F8179591dd5bA1888cf2FB3"),
+    v3_migrator_address: Some(address!("729004182cF005CEC8Bd85df140094b6aCbe8b15")),
+    nonfungible_position_manager_address: Some(address!(
+        "1238536071E1c677A632429e3655c799b22cDA52"
+    )),
+    tick_lens_address: Some(address!("d7f33bcdb21b359c8ee6f0251d30e94832baad07")),
```

Comment:
```suggestion
    tick_lens_address: Some(address!("0b343475d44EC2b4b8243EBF81dc888BF0A14b36")),
```
https://github.com/Uniswap/sdk-core/issues/127